python-package/lightgbm/sklearn.py

```diff
@@ -10,6 +10,7 @@
 import scipy.sparse
 
 from .basic import (
+    _MULTICLASS_OBJECTIVES,
```

Comment:
Could you put this in alphabetical order in this list? That'd be after `_LGBM_WeightType` below.

*(I'm not sure why `isort` didn't just do that automatically / the linter isn't complaining about that... sorry about that, I know it's no fun getting review comments like this)*

Reply:
Thank you for the review!. Because `isort` automatically sorts, it seems that this suggestion cannot be reflected 🥲.

Reply:
ah interesting, I wonder why `isort` chose to put this there 🤔 

well, not too important either way. Thanks for looking into it, and sorry for the annoying request. You've been very patient with me, I appreciate it!

Reply:
No worries at all! your requests are never annoying. Thank you for your kind words! 👏